tests/acceptance/config/behat.yml

```diff
@@ -1,20 +1,36 @@
 default:
   autoload:
-     - %paths.base%/../vendor/autoload.php
+     '': %paths.base%/../features/bootstrap
 
   suites:
     default:
```

Comment:
This suite can be called ``apiNotifications`` - behat is happy not to have a suite called default, and it will make it easier when people want to run a suite, that the suite name is always the same as the folder name of the feature files.

Reply:
will do it when moving over to drone, no need to touch travis now